app/build.gradle

```diff
@@ -112,17 +112,22 @@ def generateVersionName = { ->
 
 }
 
-
-
 android {
-    compileSdkVersion 26
-    buildToolsVersion '27.0.3'
+    compileSdkVersion 29
+
+    dexOptions {
+        javaMaxHeapSize "3g"
+    }
 
+    configurations.all {
+        resolutionStrategy.force 'com.google.code.findbugs:jsr305:1.3.9'
```

Comment:
Can you explain what this does? thanks

Reply:
https://stackoverflow.com/a/37357786/645030